qemu/tests/virtio_console.py

```diff
@@ -280,7 +280,7 @@ def test_rw_host_offline_big_data():
         guest_worker.cmd("virt.clean_port('%s'),1024" % port.name, 10)
         match, tmp = guest_worker._cmd("virt.send('%s', (1024**3)*3, True, "
                                        "is_static=True)" % port.name, 30)
-        if match is None:
+        if match is not None:
```

Comment:
Sure.
The test results of case "rw_host_offline_big_data" is all "FAIL". When I print the logs of the function _cmd(), it give the expected value "None". We hope to trigger the exception process in guest_worker._cmd() because this case is testing data transfer in disconnected situation (port closing). The command sent to the guest should fail and the code would enter the exception block, where "match" will be set with "None". The same scenario is in function test_rw_host_offline(), where "if match is not None". Thanks.